Gems/Atom/RPI/Code/Source/RPI.Private/PerformanceCVarManager.cpp

```diff
@@ -24,7 +24,7 @@ namespace AZ
             "Defines the kind of data collection and logging. If starts with 's' it will log statistical summaries, if starts with 'a' will log each sample of data (high verbosity).");
 
         AZ_CVAR(AZ::u32, r_metricsWaitTimePerCaptureBatch,
-            0, // Starts at 0, which means "do not capture performance data". When this variable changes to >0 we'll start performance capture.
+            0, // Starts at 0, which means "do not wait before starting to capture performance data".
```

Comment:
note for self: I think we should add a toolbar to allow to trigger perf capture in one click on the icon, I think that would help the reverb guys to send profiling data when they encounter perf issue within the editor